nearlib/dev.js

```diff
@@ -57,27 +58,37 @@ module.exports = {
             tempUserAccountId = 'devuser' + Date.now();
         }
         const keypair = await nearlib.KeyPair.fromRandomSeed();
-        const nearConfig = await this.getConfig();
-        const createAccount = deps.createAccount ? deps.createAccount :
+        const createAccount = this.deps.createAccount ? this.deps.createAccount :
             async (accountId, newAccountPublicKey) =>
-                createAccountWithContractHelper(nearConfig, accountId, newAccountPublicKey);
-        await createAccount(tempUserAccountId, keypair.getPublicKey());
-        localKeyStore.setKey(tempUserAccountId, keypair);
-        storage.setItem(storageAccountIdKey, tempUserAccountId);
+                this.createAccountWithContractHelper(await this.getConfig(), accountId, newAccountPublicKey);
+        await createAccount.bind(this, tempUserAccountId, keypair.getPublicKey())();
+        this.deps.keyStore.setKey(tempUserAccountId, keypair);
+        this.deps.storage.setItem(storageAccountIdKey, tempUserAccountId);
         return tempUserAccountId;
     },
     get myAccountId() {
         return window.localStorage.getItem(storageAccountIdKey);
+    },
+    getNear: function () {
```

Comment:
getter method is unnecessary, in case custom code is needed for `this.near` – it's possible to define getter similar to `myAccountId`.

Reply:
not used, removed